packages/components-js/projects/components-manager-core/src/services/library-handler.ts

```diff
@@ -26,15 +26,15 @@ export type LoadComponentLibraryOptions = {
  * @param options - LoadComponentLibraryOptions
  */
 export function loadComponentLibrary({ script, version, prefix }: LoadComponentLibraryOptions): void {
-  const data = getLibraryHandlerData(version);
+  const data = getLibraryHandlerData(version as `${number}.${number}.${number}`);
   const { isInjected, prefixes = [], registerCustomElements } = data;
 
   const [collidingVersion] = Object.entries(getComponentsManagerData()).filter(
-    ([v, cmData]) => v !== version && cmData.prefixes.includes(prefix)
+    ([v, cmData]) => v !== version && typeof cmData !== 'string' && cmData.prefixes.includes(prefix)
   );
   if (collidingVersion) {
     throw new Error(
-      `[Porsche Design System] prefix '${prefix}' is already registered with version '${collidingVersion[0]}' of the Porsche Design System. Please use a different one.
+      `[Porsche Design System v${version}] prefix '${prefix}' is already registered with version '${collidingVersion[0]}' of the Porsche Design System. Please use a different one.
```

Comment:
```suggestion
      `[Porsche Design System] v${version} with prefix '${prefix}' is already registered with version '${collidingVersion[0]}' of the Porsche Design System. Please use a different one.
```

Not sure, if it's good to have the version within the `[]` because then we have another inconsistency within the log output compared to other logs.

Reply:
We usually always have the version in the brackets so it is possible to find out which prefix and team is causing it.
<img width="450" alt="Bildschirmfoto 2023-08-02 um 10 41 25" src="https://github.com/porsche-design-system/porsche-design-system/assets/4380295/d5965928-e209-41f5-93be-8e587ec33389">